oauth2/oauth2-resource-server/src/main/java/org/springframework/security/oauth2/server/resource/web/server/ServerBearerTokenAuthenticationConverter.java

```diff
@@ -38,20 +38,22 @@
  * from the {@link ServerWebExchange}.
  *
  * @author Rob Winch
- * @since 5.1
```

Comment:
Please leave this line on here. Thanks!

Reply:
Good spot. I missed that when committing.